README.md

```diff
@@ -29,12 +34,24 @@ OpsGenie. It also takes care of silencing and inhibition of alerts.
 
 ## C
 
+### Cardinality
+
+Cardinality refers to the number of unique values in a set of data. In the context of monitoring systems, high cardinality can negatively impact performance, so it's important to manage the cardinality of metrics and labels
+
 ### Counter
 
 As the name says, counters are used to monitor how frequently an event occurs within a program.
 They help monitor metrics that increase monotonically and are exposed as time series.
 For example, total requests to an API endpoint is a counter metric, it keeps increasing over time.
 
+### CI
+
+CI (Continuous Integration) is a practice of integrating code changes into a shared repository frequently. CI helps detect and resolve conflicts early and ensures that new code integrates smoothly with the existing codebase.
```

Comment:
```suggestion
CI (Continuous Integration) integrates code changes into a common branch of a shared repository and runs automated tests against it every time the changes are merged, or a commit is made. This allows for detecting issues in the code faster because it is tested continuously, as the name says.
```